javaslang/src/main/java/javaslang/$.java

```diff
@@ -42,6 +42,10 @@
     static <T1, T2, T3, T4, T5, T6, T7> Tuple7<T1, T2, T3, T4, T5, T6, T7> Tuple7(Tuple7<T1, T2, T3, T4, T5, T6, T7> tuple7) { return tuple7; }
     @Unapply
     static <T1, T2, T3, T4, T5, T6, T7, T8> Tuple8<T1, T2, T3, T4, T5, T6, T7, T8> Tuple8(Tuple8<T1, T2, T3, T4, T5, T6, T7, T8> tuple8) { return tuple8; }
+    @Unapply
```

Comment:
In a future Javaslang the patterns will be part of the API class if possible. Then the methods will be generated.

**Currently I search a way for the following (targeted for 2.1.0):**

$.java will be deprecated (including Patterns.java, which is generated by the annotation processor). The core project 'javaslang' will not depend on javaslang-match any more (currently only compile-time). The patterns will be inlcuded in API, e.g.

* Tuple(1, 2) should create a Tuple
* Tuple($(1), $(2)) should create a Pattern that matches a Tuple

Also some patterns might change. E.g. Cons(head, tail) and Nil() instead of List(head, tail) and List().